lib/active_model/serializer.rb

```diff
@@ -5,7 +5,7 @@
 require 'active_model/serializer/configuration'
 require 'active_model/serializer/fieldset'
 require 'active_model/serializer/lint'
-require 'active_model/serializer/utils'
+require 'active_model/serializer/include_tree'
```

Comment:
`IncludedAttributesTree` clearly does not work as this is currently used only for relationships. Moreover, it's general enough so it could virtually be used for representing any arborescent structure with wildcards. I don't mind changing the name, though.